Lab6.ipynb

```diff
@@ -0,0 +1,692 @@
6oK+RaWx52QDkkbcEcCewDkAVfXjJA8cbEiSJHXXyzqFjwEOB06gWcj+PcDfzXZSkhOTXJPkBx1lxyS5MsmF7WPfjn1HJ1mb5EdJnt1RvndbtjbJUXN5c5IkDcitVXXb5EaSzWiWdZIkaej00lL4p8BDOiu3Hn0c+ABw8pTyv6+quyWVSfYADgYeBTwY+HqSh7W7Pwg8E1gHnJtkVVVdMsdYJElaSGcleRNw7yTPBP43zdh8SZKGTi8thd8DtpnrhavqW8B1PR5+AHBKVd1aVT8F1tJ0u9kTWFtVl7VJ6SntsZIkDbOjaNb1/T7wCmAN8JaBRiRJ0jR6aSl8EPDDJOdy9zGFG7okxZFJXgqcB7yuqq6nGYB/dscx69oygCumlO/V7aJJVtAM6mfp0qUbGJokSRuvXcD+JJoxhQX8qKrsPipJGkq9JIVvm8fX+xBwHE0FeRzN+MSXA+lybNG9JbNrpVpVK4GVABMTE1a8kqSBSbIfzWyjP6Gp43ZL8oqq+upgI5Mk6Z5mTQqr6qz5erGq+sXk8yQfAb7cbq4Ddu44dCfgqvb5dOWSJA2r9wB/UlVrAZI8FPgKYFIoSQMydXb7y0/Yb0CRDJ9ZxxQmuTnJTe3jN0nuSHLThrxYkh06Nv8UmJyZdBVwcJItkuwG7A58BzgX2D3JbknuRTMZzaoNeW1JkhbQNZMJYesy4JpBBSNJ0kx6aSm8X+d2kufSTAAzoySfAZ4KPCDJOppuqE9N8jiaLqCX0wy+p6ouTnIqcAlwO3BEVd3RXudI4HRgU+DEqrq41zcnSdJCSnJg+/TiJGuAU2nqvOfT3OgcG96RlzTsuq2LPa6fVb2MKbybqvpiL+sFVtULuxR/dIbj3wG8o0v5GppZ2yQtIv7DqEVqecfzXwBPaZ+vB7Zd+HAkSZrdrElhx11PaLqbTuACvJIk3UNVHTroGIZVtzvykqTh0EtLYeddz9tpun26VqAkSdNox8e/CtiVjrp2I5ZzkiSpb3oZU+hdT0mS5uaLNEMmVgN3DjgWSZJmNG1SmOStM5xXVXVcH+LRmLJbkaRF5jdV9f5BByFJ42xD/r+c7ZzFOgfCTC2Fv+pSthVwGHB/msXnJUnSPb0vyduArwG3ThZW1QXz+SJJHgK8GVhSVQfN57UlSeNj2qSwqt4z+TzJ/YBXA4cCp9AsyitJkrp7DPAS4Gnc1X202u0ZJTkReA7NWoeP7ijfG3gfzRJN/1xVJ1TVZcBhSU6b5/glSWNkxjGFSbYDXgu8GDgJeEJVXb8QgUmSNML+FHhIVd22Aed+HPgAcPJkQZJNgQ8CzwTWAecmWVVVl8xDrJKkMbfJdDuS/C3NQrs3A4+pqmNMCCVJ6sn3gG025MSq+hZw3ZTiPYG1VXVZm2iegjOBS5LmyUwtha+jGQfxFuDNSSbLQzPRzNZ9jk3SGOk2sHuxDubWWHgQ8MMk53L3MYUbuiTFjsAVHdvrgL2S3B94B/D4JEdX1d90OznJCmAFwNKlSzcwBEnSYjXTmMJpWxElSdKM3jbP10uXsqqqXwKvnO3kqloJrASYmJioeY5NkjTielm8XpIkzUFVnTXPl1wH7NyxvRNw1Ty/hiRpTNkaKEnSPEtyc5Kb2sdvktyR5KaNuOS5wO5JdktyL+BgYNUcY1qeZOWNN964EWFIkhYjk0JJkuZZVd2vqrZuH1sCz6OZUXRWST4DfBt4eJJ1SQ6rqtuBI4HTgUuBU6vq4jnGtLqqVixZsmRub0aStOjZfVQLrtuEIpK0mFXVF5Mc1eOxL5ymfA2wZl4DkyQJk0JJkuZdkgM7NjcBJmgWr5ckaeiYFEqSNP+Wdzy/HbicAa8rmGQ5sHzZsmWDDEOSNIRMCiVJmmdVdeigY5iqqlYDqycmJg4fdCySpOFiUihJ0jxJ8tYZdldVHbdgwUiS1COTQklDa+qkRJefsN+AIpF69qsuZVsBhwH3B0wKJUlDx6RQkqR5UlXvmXye5H7Aq4FDgVOA90x33kKYzzGF3WaR9qaNJI0u1ymUJGkeJdkuyfHARTQ3X59QVW+sqmsGGZfrFEqSpmNLoSRJ8yTJ3wIHAiuBx1TVLQMOSZKkWfWtpTDJiUmuSfKDjrLtkpyR5Mft123b8iR5f5K1SS5K8oSOcw5pj/9xkkP6Fa8kSfPgdcCDgbcAVyW5qX3cnOSmAccmSVJX/Wwp/DjwAeDkjrKjgG9U1QlJjmq33wjsA+zePvYCPgTslWQ74G3ctejv+UlWVdX1fYxbkqQNUlVDOyyj3+sUdhtnKEnjYDFMjNe3yquqvgVcN6X4AOCk9vlJwHM7yk+uxtnANkl2AJ4NnFFV17WJ4BnA3v2KWZKkxcoxhZKk6Sz0Hc0HVdXVAO3XB7blOwJXdBy3ri2brlySJEmSNA+GpZtLupTVDOX3vECyIsl5Sc5bv379vAYnSZIkSYvVQieFv2i7hdJ+nZyeex2wc8dxOwFXzVB+D1W1sqomqmpi++23n/fAJUmSJGkxWuikcBUwOYPoIcCXOspf2s5C+iTgxrZ76enAs5Js285U+qy2TJIkSZI0D/o2+2iSzwBPBR6QZB3NLKInAKcmOQz4GfD89vA1wL7AWuDXwKEAVXVdkuOAc9vjjq2qqZPXSJKkWfR79lFJGgeLdablviWFVfXCaXY9vcuxBRwxzXVOBE6cx9AkjajFMOWzNChVtRpYPTExcfigY5EkDZdhmWhGkiRJkjQAJoWSJEmSNMb61n1UmrRY+15LkiRJi4EthZIkSZI0xkwKJUkaA0mWJ1l54403DjoUSdKQMSmUJGkMVNXqqlqxZMmSQYciSRoyJoWSJEmSNMacaEbSyOo2iZFrF0qSJM2NSaHmnbONSpIkSaPD7qOSJEmSNMZMCiVJkiRpjJkUSpIkSdIYMymUJEmSpDHmRDOSJI2BJMuB5cuWLRt0KJI0VkZhtnRbCiVJGgMuXi9Jmo4thdooLj8hSZIkjTZbCiVJkiRpjNlSKGlRmdp6
```

Comment:
They would be exactly the same if you built them on the exact background you generated in problem 1.